format/private/format.sh

```diff
@@ -94,9 +94,20 @@ function ls-files {
 
         # TODO: determine which staged changes we should format; avoid formatting unstaged changes
         # TODO: try to format only modified regions of the file (where supported)
-        git ls-files --cached --modified --other --exclude-standard "${patterns[@]}" "${patterns[@]/#/*/}" | {
-          grep -vE "^$(git ls-files --deleted)$" || true;
-        }
+        files=$(git ls-files --cached --modified --other --exclude-standard "${patterns[@]}" "${patterns[@]/#/*/}" | {
+          grep -vE \
+            "^$(git ls-files --deleted)$" \
+          || true;
+        })
+        non_ignored_files=()
+        for file in "${files[@]}"; do
+            if [[ $(git check-attr -a -- $file | grep "rules_lint_ignored: true" ) == "" ]]; then
```

Comment:
is this parse-brittle? What if my `.gitattributes` had no space like `rules_lint_ignored:true` ?

Reply:
It wasn't brittle before, but its more idiomatic now since I just check if the attribute is set or unset rather than for a specific value